conjur-api/Client.cs

```diff
@@ -139,7 +147,7 @@ public WebRequest Request(string path)
         /// authorization header set using <see cref="Authenticator"/>.</returns>
         public WebRequest AuthenticatedRequest(string path)
         {
-            return ApplyAuthentication(Request(path));
+            return ApplyAuthentication(Request(path + ((m_acting_as != null) ? $"&acting_as={WebUtility.UrlEncode(m_acting_as)}" : String.Empty)));
```

Comment:
You may be checking for this somewhere already and I just can't locate it, but currently this assumes you already have at least one query parameter in the URL. 

It looks like you should check here whether the URL already contains a query parameter  or not, and then prepend either the `?` or `&` as appropriate.

Reply:
You are correct, acting_as support was develop for ListResources case in which we have previous query params configured. If someone decide to use (public) AuthenticatedRequest directly supplying a path without query params it will return undefined response. We can write a code that will handle this use case using String path - I hope that in the future we will refactor this code and pass URI instead of String in which you can enquire the existing of params via methods...